derp/derp_server.go

```diff
@@ -141,6 +142,11 @@ type Server struct {
 	// because it includes intra-region forwarded packets as the
 	// src.
 	sentTo map[key.Public]map[key.Public]int64 // src => dst => dst's latest sclient.connNum
+
+	addrOfKey map[netaddr.IPPort]key.Public
```

Comment:
If you're going to use "Of" in the name, a `map[A]B` is named `bOfA`, so `keyOfAddr`. The alternative is `addrKey` which can be read like "addr's key".